providers/ethereum-provider/src/EthereumProvider.ts

```diff
@@ -602,25 +608,28 @@ export class EthereumProvider implements IEthereumProvider {
     if (this.rpc.showQrModal) {
       let appKit;
       try {
-        const { createAppKit } = await import("@reown/appkit/core");
-        const { convertWCMToAppKitOptions } = await import("./wcmToAppKit");
-        const options = convertWCMToAppKitOptions({
-          ...this.rpc.qrModalOptions,
-          chains: [...new Set([...this.rpc.chains, ...this.rpc.optionalChains])],
-          metadata: this.rpc.metadata,
-          projectId: this.rpc.projectId,
-        });
+        if (isBrowser()) {
```

Comment:
What happens on mobile? 
Does this affect mobile apps with web views? Or are web views considered in is browser?

Reply:
it works fine on mobile, its widely used around the SDK